src/common/utils.ts

```diff
@@ -1,14 +1,32 @@
 import engine from 'store/src/store-engine';
 import lclStorage from 'store/storages/localStorage';
+import { c32addressDecode } from 'c32check';
 
 export const store = engine.createStore([lclStorage]);
 
+/**
+ * validateStacksAddress
+ *
+ * @param {String} stacksAddress - the STX address to validate
+ */
+const validateStacksAddress = (stacksAddress: string) => {
```

Comment:
Wouldn't a `(stxAddress: string) => boolean` signature be more helpful here? Rather than true or throws error?

Reply:
Yes, I just pulled this from stacks-utils, happy to update it